src/components/ContactsPage.js

```diff
@@ -29,6 +37,31 @@ class ContactsPage extends React.Component {
     );
   }
 
+  changeIsChecked(id) {
+    this.setState((prevState) => {
+      const contacts = [...prevState.contacts];
+      const contact = contacts.find(item => item.id === id);
+      contact.isChecked = !contact.isChecked;
+      return { contacts };
+    });
+  }
+  
+  increaseLikes(id) {
```

Comment:
you can merge these 2 methods: increaseLikes and decreaseLikes into one: updateLikes(id, likes) and pass 1 and -1 as params